Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/SubmissionRepository.kt

```diff
@@ -57,6 +57,7 @@ class SubmissionRepository @Inject constructor(
 
     // to be used by new submission flow screens
     val hasGivenConsentToSubmission = submissionSettings.hasGivenConsent.flow
+    val hasViewedTestResult = submissionSettings.hasViewedTestResult.value
```

Comment:
Why is that not a flow?

Reply:
I originally had it as flow, and read it in SubmissionCardState as: 
    private val testViewed = submissionRepository.hasViewedTestResult.asLiveData()

But the value kept returning null. Perhaps implementation was incorrect? I can have a look again.

Reply:
LiveData needs an observer to emit data. We should expose the flow in the repository, otherwise it is not observable.